doc/tutorials/imgproc/morph_lines_detection/moprh_lines_detection.md

```diff
@@ -0,0 +1,87 @@
+Extract horizontal and vertical lines by using morphological operations {#tutorial_moprh_lines_detection}
+=============
+
+Goal
+----
+
+In this tutorial you will learn how to:
+
+-   Apply two very common morphology operators (i.e. Dilation and Erosion), with the creation of custom kernels, in order to extract straight lines on the horizontal and vertical axes. For this purpose, you will use the following OpenCV functions:
+    -   @ref cv::erode
+    -   @ref cv::dilate
+    -   @ref cv::getStructuringElement
+
+    in an example where your goal will be to extract the music notes from a music sheet.
+
+Theory
+------
+
+### Moprhology Operations
+@brief Morphology is a set of image processing operations that process images based on predefined *structuring elements* known also and as kernels. The value of each pixel in the output image is based on a comparison of the corresponding pixel in the input image with its neighbors. By choosing the size and shape of the kernel, you can construct a morphological operation that is sensitive to specific shapes regarding the input image.
+
+Two of the most basic morphological operations are dilation and erosion. Dilation adds pixels to the boundaries of the object in an image, while erosion does exactly the opposite. The ammount of pixels added or removed, respectively depends on the size and shape of the structuring element used to process the image. In general the rules followed from these two operations have as follows:
+
+
+-   **Dilation**: The value of the output pixel is the ***maximum*** value of all the pixels that fall within the structuring element's size and shape. For example in a binary image, if any of the pixels of the input image falling within the range of the kernel is set to the value 1, the corresponding pixel of the output image will be set to 1 as well. The latter applies to any type of image (e.g. grayscale, rgb, etc).
```

Comment:
You've read too far :smiley:, I meant only the _"Use of asterisks"_ section. 
To avoid __possible__ issue just use `_` instead of `*` to make text bold or italic.